### DISCUSSION TITLE
0.5.3

### DISCUSSION THREAD

**Discussion by z-huang** [z-huang/music#1024] [Announcements]
- Better UI
- Grid layout for albums and playlists
- Minor enhancement and bug fixes

[Compare versions](https://github.com/z-huang/InnerTune/wiki/App-Versions)

<hr /><em>This discussion was created from the release <a href='https://github.com/z-huang/InnerTune/releases/tag/v0.5.3'>0.5.3</a>.</em>

**Comment by darkza** [comment]
Hey! is the Touchscreen swipe to next title removed? Its not working anymore. Since 5.0 i 🤔.

**Reply by z-huang** [reply]
See #1017.

**Reply by zippostar** [reply]
Please support SD card 😘👍

**Comment by netrunner-exe** [comment]
> Hey! is the Touchscreen swipe to next title removed? Its not working anymore. Since 5.0 i 🤔.

There have been several releases already, but it has not been fixed. Let's hope that maybe by the next release it will be possible to return the swipe.

**Comment by arc1e** [comment]
Is there an option within app setting to force Android Auto skin mode?
Android Auto mode finds it easier to operate vs the standard mode.

Login? Pipe not work? Logging into Google Account on 3rd party app is uncomfortable.

Overall, definitely good to see YTMusic alternatives is available.

**Reply by z-huang** [reply]
What is Android Auto skin mode?

Login is safe. Since the source code is available in GitHub and everyone can check the code, you can login without worries.

**Comment by arc1e** [comment]
Android Auto mode looks different and simple to operate.I've been using your app on my Android dongle for my car. Runs like the standard app on a phone. I hope there is a way to change the App design to Android Auto mode instead.Sent from my iPhoneOn Sep 7, 2023, at 20:19, Zion Huang ***@***.***> wrote:﻿
What is Android Auto skin mode?
Login is safe. Since the source code is available in GitHub and everyone can check the code, you can login without worries.

**Reply by darkza** [reply]
Android Auto works!
There is no Theme or UI for your Phone that it looks like on your car display.

i don't even think that this is a good idea because you cant search for a song on your android Auto Ui on your car display. This is why i dont use Android Auto! the real app is more confi on my atoto android Head unit!

**Comment by arc1e** [comment]
When used in a car,Intuitive & Easy Navigation is to prevent Driving Accidents.Android Auto version of YT Music is very limited. But their Button are HUGE easy to access.Yours is Small to Access. Just what if, you can make it Bigger to Operate. This will make it Fun to use. And Safer.Sent from my iPhoneOn Sep 8, 2023, at 01:37, darkza ***@***.***> wrote:﻿
Android Auto works!
There is no Theme or UI for your Phone that it looks like on your car display.
i don't even think that this is a good idea because you cant search for a song on your android Auto Ui on your car display. This is why i dont use Android Auto! the real app is more confi on my atoto android Head unit!

**Comment by PaulReg** [comment]
The layout in this version is really nice. But, could we also have a "downloaded" tag in albuns, just like the one in music section?

**Comment by arc1e** [comment]
After logging into my YT Music.Noticed it pulls My Library..However, everything in there are my “old library”. Everything new, don’t exist?!Looks like it is a built in Android KitKat Browser accessing YT Music.US Army VeteranSent from my iPhoneOn Sep 12, 2023, at 16:57, PaulReg ***@***.***> wrote:﻿
The layout in this version is really nice. But, could we also have a "downloaded" tag in albuns, just like the one in music section?

**Comment by Yonle** [comment]
Sometime the queue is not empty, and sometime it is empty.

This happens since the JetBrain update.
![Screenshot_20230918-195318](https://github.com/z-huang/InnerTune/assets/63401712/3294c321-d46c-4b04-8e76-9f1304dd6470)

**Comment by DanielJLBarbosa** [comment]
Hey Bro, I'm just here to thank you for all your work with InnerTune. Honestly, this is the best music streaming there is, in my opinion. Thank you very much for your work!! And I will even make a cash contribution to you. See you later and be well!!

**Comment by arc1e** [comment]
That Dice Button!Be great if that button can be moved.Such as a Floating button on top of the app.US Army VeteranSent from my iPhoneOn Sep 15, 2023, at 17:00, Andy Wen ***@***.***> wrote:﻿After logging into my YT Music.Noticed it pulls My Library..However, everything in there are my “old library”. Everything new, don’t exist?!Looks like it is a built in Android KitKat Browser accessing YT Music.US Army VeteranSent from my iPhoneOn Sep 12, 2023, at 16:57, PaulReg ***@***.***> wrote:﻿
The layout in this version is really nice. But, could we also have a "downloaded" tag in albuns, just like the one in music section?

**Comment by Zenith0xYT** [comment]
TY INNERTUNE NOW I HAVE AD FREE MUSIC 🥰🥰🥰🥰

**Comment by arc1e** [comment]
Ads free without subscription,Once Google algorithm catches up..Account will be flagged and banned* Irreversible *On Nov 7, 2023, at 06:51, TheBothYT ***@***.***> wrote:﻿
TY INNERTUNE NOW I HAVE AD FREE MUSIC 🥰🥰🥰🥰

**Comment by 7tyzh** [comment]
When to update? 🥵

**Comment by sharoon7171** [comment]
I have downloaded songs offline and added in library
but album images are not song images are not downloaded

and because of that without internet connection no images are loaded and when there's no image loaded then music player crashes when we press next song button on lock screen or in player

please add functionality so we can use the app without internet connection

**Comment by NightSun29** [comment]
Please option to add an entire album to library

**Reply by RenautMestdagh** [reply]
And entire playlist

**Comment by BeInvisible07** [comment]
![IMG_20231129_145349](https://github.com/z-huang/InnerTune/assets/152383072/080e9c36-9dfd-4ff8-81fb-1aa184046a5a)
Artist Album songs are not showing #740 #487

**Reply by netrunner-exe** [reply]
Singles too...
![Screenshot_2023-11-29-18-57-32-076_com zionhuang music](https://github.com/z-huang/InnerTune/assets/81887288/d47149b1-cfb1-4071-b81e-853bd92f6426)

**Comment by AbielDINo** [comment]
How can I use it on iPhone 11

**Reply by RenautMestdagh** [reply]
You can't. Apple doesn't let you sideload apps.

**Reply by BennyDioxide** [reply]
Get a Mac (or a VM, apple isn't happy), make it a iOS app using Compose Multiplatform, and "test" on your phone
But remember, it's always easier said than done.

**Comment by arc1e** [comment]
@AbielDino, note it’s an APK means iPhones are no go. It is built for Android.US Army VeteranSent from my iPhoneOn Dec 15, 2023, at 10:32, AbielDINo ***@***.***> wrote:﻿
How can I use it on iPhone 11

**Comment by zippostar** [comment]
Please updates 😘

**Comment by chamudha** [comment]
Showing bigger icons in notification ![Screenshot_2024-01-06-09-02-47-445_com.zionhuang.music.jpg](https://github.com/z-huang/InnerTune/assets/134184550/9f38cb6b-b5ab-4995-bf26-573ce70c76bf)

**Comment by Hermez-09** [comment]
Excellent project, only some details need to be polished such as: 

-The name of some artists. Sometimes they tend to change according to the song. 

-When I type %5 or 5%, it takes me out of the application.  

-It does not fully load the videos, singles or albums. 

-And when playing offline, it pauses, even if it is in the download list. 

Those would be some details that I have seen but, anyway, your work is appreciated.

**Comment by chamudha** [comment]
Can add tab title mode and animation icon in home screen of retro music app for innertune appearance ??![Screenshot_2024-02-06-10-33-11-873_code.name.monkey.retromusic.jpg](https://github.com/z-huang/InnerTune/assets/134184550/712f3973-e020-4b56-b1ac-c51ec161d7ea)

https://github.com/z-huang/InnerTune/assets/134184550/4f12a98a-5d92-4143-b58d-9e65f6ec7311

**Comment by firestylefirebee** [comment]
The feature of exporting the sings manually witht he song file seems to be removed, please bring backt he feature of exporting.

**Comment by qq590059** [comment]
Thanks to the author. 
Huawei mobile phone crashes when running the app. 
Please solve it. Thank you.

**Reply by sharoon7171** [reply]
for me on huawei it crashes when we use without internet
album arts do not load even for downloaded songs and then it crashes

**Comment by Gaurav1137** [comment]
![Screenshot_2024-02-27-14-18-20-51](https://github.com/z-huang/InnerTune/assets/91793726/8c43d2f7-b3c8-4aaa-bf37-8f30e53ce239)

Playlist and song doesn't lode
Fix this 
Please 🙏

**Comment by chamudha** [comment]
https://github.com/z-huang/InnerTune/assets/134184550/04d41c96-2aa6-4d34-b46f-7361c5aadb6f



Can add a sorting method like this for artist page ?

**Comment by KevinLin-Taiwan** [comment]
It can't play music and loading the music list now!!! I tried to delete and reload the  InnerTune_v0.5.3_full_arm64-v8a.apk 
![Screenshot_2024-06-22-02-34-44-607_com zionhuang music](https://github.com/z-huang/InnerTune/assets/173496608/8b148643-83dd-478d-8521-6234c6e10c97)
again but still doesn't work......Please make a new update version🙏😫

**Comment by JLynn-LY** [comment]
need update

**Comment by CesarF08** [comment]
As of today, June 30, 2024, playlists do not load when logging into the account or trying to open any created by YouTube Music. Everything else is working great. Hopefully, this can be fixed soon. Thank you.

A día de hoy 30 Junio de 2024, no cargan las playlist al momento de iniciar sesión en la cuenta o intentar abrir alguna de las son creadas por YouTube music, de resto todo sigue funcionando genial, ojalá pueda ser corregido pronto, gracias

**Reply by fanieko** [reply]
You can always try the [malopieds fork](https://github.com/Malopieds/InnerTune), but there some design changes that you may not like..

**Reply by obitopy** [reply]
![Screenshot_20240707_085833_com brave browser](https://github.com/z-huang/InnerTune/assets/173553435/7ed96dfc-bb2c-47e5-8539-6e89b4401f2e)
good virus

**Comment by chamudha** [comment]
Can add a option to play next and previous song by swiping mini player ?

https://github.com/z-huang/InnerTune/assets/134184550/1b6cf2b2-7c6f-4042-b56e-fa90d1a0380f

**Comment by marcorizzo97** [comment]
With version 0.5.3 full no albums are displayed or loaded, it tries to do an upload but to no avail. Solve this problem and various problems in the app with a new update.
